### PR TITLE
Add return to avoid double-consent

### PR DESCRIPTION
**Pull request by mserve**
See issue #330

### CODE REVIEW

File: openvidu-browser/src/OpenVidu/OpenVidu.ts [hunk 1]
```diff
@@ -946,4 +947,4 @@ export class OpenVidu {
     }
   }
 
-}
\ No newline at end of file
+}
```
Comment (pabloFuente) [inline openvidu-browser/src/OpenVidu/OpenVidu.ts]: Please, remove this new line char
```suggestion
```

Comment (mserve) [reply openvidu-browser/src/OpenVidu/OpenVidu.ts]: Done, seems to be a bug in the GiHib online editor....

Comment (pabloFuente) [reply openvidu-browser/src/OpenVidu/OpenVidu.ts]: Last closing bracket has been removed. It was only supposed to be the last new line character, not last bracket. If you don't mind I'll commit the change myself to integrate it right now on the master branch, as new release is coming soon.

Comment (mserve) [reply openvidu-browser/src/OpenVidu/OpenVidu.ts]: ok, perfext, thx!

**Pull request merged**